community/spl402-starter-kit/server/index.ts

```diff
@@ -0,0 +1,130 @@
+import express, { Request, Response } from 'express'
+import cors from 'cors'
+import dotenv from 'dotenv'
+import { createServer, createExpressMiddleware } from 'spl402'
+
+dotenv.config()
+
+const app = express()
+const PORT = process.env.PORT || 3001
+
+app.use(
+  cors({
+    origin: process.env.FRONTEND_URL || '*',
+    credentials: true,
+    exposedHeaders: ['X-Payment-Required'],
+  }),
+)
+
+app.use(express.json())
+
+const spl402 = createServer({
+  network: 'mainnet-beta',
```

Comment:
Should be a parameter like the rpcUrl to match the configuration of the user. 

I'm not sure we should default to mainnet. Users testing this out will prefer to work on devnet first.